conformance/conformance.proto

```diff
@@ -57,11 +57,13 @@ option java_package = "com.google.protobuf.conformance";
 //   2. parse the protobuf or JSON payload in "payload" (which may fail)
 //   3. if the parse succeeded, serialize the message in the requested format.
 message ConformanceRequest {
+  string test_name = 1;
```

Comment:
Yeah, let's take it out for now and add it when it's actually used by some tested programs. For protocols and interfaces I lean towards keeping things simple unless there are clear usages of the additions. I'm not convinced that the test name would be useful in the in-process case as well. If the runner and the tested code is in the same process and both of them know the test case name, it's weird to think that only the tested code can log the test case name while the runner cannot.